composer.json

```diff
@@ -28,7 +28,7 @@
     "require": {
         "php": ">=5.4",
         "ramsey/uuid": "^3.0",
-        "doctrine/dbal": "^2.5"
+        "doctrine/orm": "^2.5"
```

Comment:
Since `doctrine/orm` requires `doctrine/dbal`, I just change this requirement rather than having both.